engine/class_modules/priest/sc_priest.hpp

```diff
@@ -583,8 +583,8 @@ struct priest_t final : public player_t
   void trigger_psychic_link( action_state_t* );
   void trigger_wrathful_faerie();
   void remove_wrathful_faerie();
-  int shadow_weaving_active_dots( const player_t* target ) const;
-  double shadow_weaving_multiplier( const player_t* target ) const;
+  int shadow_weaving_active_dots( const player_t* target, const unsigned int id ) const;
```

Comment:
maybe rename this to action_id or spell_id, otherwise it is not clear what id you want here without looking at the implementation.